frontend/src/features/admin-form/create/featureTour/FeatureTourTooltip.tsx

```diff
@@ -66,8 +72,11 @@ export const FeatureTourTooltip = ({
         alignItems="center"
         justifyContent="space-between"
       >
-        {/* Todo: Create pagination component */}
-        <Text>Pagination Component</Text>
+        <ProgressIndicator
+          numIndicators={FEATURE_STEPS.length}
+          currActiveIdx={index}
+          onClick={paginationCallback}
+        />
         {isLastStep ? (
           <Button {...skipProps}>Done</Button>
```

Comment:
May I clarify when you mean "can just use primary props here", you mean add the primary props to the `ProgressIndicator` component?

Reply:
There is no need to add `continuous` prop because we are handling the transitioning of steps from 1 step to another via a controlled state (`stepIndex`). We are updating that state through the custom callback.

Reply:
gotcha, ignore this then.